evm-test-helpers/src/subproviders/ovm-ganache.ts

```diff
@@ -0,0 +1,50 @@
+import { GanacheProvider, JSONRPCRequestPayload } from 'ethereum-types'
+// @ts-ignore
+import Ganache from 'ganache-core'
+import { Subprovider, Callback, ErrorCallback } from '@0x/subproviders'
+import { makeDebug } from '../debug'
+const ovm = require('@eth-optimism/ovm-toolchain')
+
+const log = makeDebug('ovm-ganache-subprovider')
+
+/**
+ * This class implements the [web3-provider-engine](https://github.com/MetaMask/provider-engine) subprovider interface.
+ * It intercepts all JSON RPC requests and relays them to an in-process ganache instance.
+ */
+export class OVMGanacheSubprovider extends Subprovider {
+  private readonly _ganacheProvider: GanacheProvider
```

Comment:
No need for underscores, we know its private :D
```suggestion
  private readonly ganacheProvider: GanacheProvider
```

Reply:
[0x/subproviders](https://github.com/0xProject/0x-monorepo/blob/development/packages/subproviders/src/subproviders/ganache.ts) had this kind of convention so we just took it from there.